### ISSUE TITLE
[Bug]: The response of `list backup` is `{'requestId': '26340f9f-8291-11ed-88ec-6045bdae15be', 'code': 3, 'msg': 'not found'}` after creating backup

### ISSUE THREAD

**Issue by zhuwenxing** [zilliztech/milvus-backup#57]
### Current Behavior

```
[2022-12-23 07:12:19 - INFO - ci_test]: create backup response: {'requestId': '232fe47b-8291-11ed-88ec-6045bdae15be', 'msg': 'create backup is executing asynchronously', 'data': {'id': '23300c0c-8291-11ed-88ec-6045bdae15be', 'name': 'backup_x8FhJ0AW'}} (test_create_backup.py:52)
[2022-12-23 07:12:24 - INFO - ci_test]: backup backup_x8FhJ0AW is ready in 5.049671649932861 seconds (milvus_backup.py:66)
[2022-12-23 07:12:24 - INFO - ci_test]: list backup response: {'requestId': '26340f9f-8291-11ed-88ec-6045bdae15be', 'code': 3, 'msg': 'not found'} (test_create_backup.py:57)
```

### Expected Behavior

The response of `list backup` should return the right backups

### Anything else?

```
[2022-12-23 07:11:45 - INFO - ci_test]: create backup response: {'requestId': '0ed37b7f-8291-11ed-bd61-002248a6f8c2', 'msg': 'success', 'data': {'id': '0ed39f5a-8291-11ed-bd61-002248a6f8c2', 'state_code': 2, 'name': 'backup_KQLLsowq', 'backup_timestamp': 1671779504897, 'collection_backups': [{'collection_name': 'create_backup_dZjB4oVF', 'backup_timestamp': 438246966296576}]}} (test_create_backup.py:52)
[2022-12-23 07:11:45 - INFO - ci_test]: list backup response: {'requestId': '0ee594b5-8291-11ed-bd61-002248a6f8c2', 'code': 3, 'msg': 'not found'} (test_create_backup.py:57)
------------- generated html file: file:///tmp/ci_logs/report.html -------------
=========================== short test summary info ============================
FAILED testcases/test_create_backup.py::TestCreateBackup::test_milvus_create_backup[float-1-False] - AssertionError: assert 'backup_KQLLsowq' in []
```

failed job: https://github.com/zilliztech/milvus-backup/actions/runs/3763895849/jobs/6397810361
log:
https://github.com/zilliztech/milvus-backup/suites/10025422439/artifacts/487361838

**Comment by zhuwenxing** [comment]
```
2023-01-11T08:47:57.089Z] =========================== short test summary info ============================

[2023-01-11T08:47:57.089Z] FAILED testcases/test_create_backup.py::TestCreateBackup::test_milvus_create_backup[binary-1-False] - AssertionError: assert 'backup_vqChfrEu' in []

[2023-01-11T08:47:57.089Z] FAILED testcases/test_delete_backup.py::TestDeleteBackup::test_milvus_delete_backup[1-False] - AssertionError: assert False

[2023-01-11T08:47:57.089Z]  +  where False = <built-in method issubset of set object at 0x7f04664d52e0>(set())

[2023-01-11T08:47:57.089Z]  +    where <built-in method issubset of set object at 0x7f04664d52e0> = {'backup_CL5TZRxx'}.issubset

[2023-01-11T08:47:57.089Z]  +      where {'backup_CL5TZRxx'} = set(['backup_CL5TZRxx'])

[2023-01-11T08:47:57.089Z]  +    and   set() = set([])

[2023-01-11T08:47:57.089Z] FAILED testcases/test_create_backup.py::TestCreateBackup::test_milvus_create_backup[all-2-True] - AssertionError: assert 'backup_KRPxzT08' in []

[2023-01-11T08:47:57.089Z] =================== 3 failed, 88 passed in 272.36s (0:04:32) ===================
```


failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/milvus_backup_api_test/detail/milvus_backup_api_test/25/pipeline

log:
[artifacts-backup-test-25-server-logs (1).tar.gz](https://github.com/zilliztech/milvus-backup/files/10390476/artifacts-backup-test-25-server-logs.1.tar.gz)
[artifacts-backup-test-25-pytest-logs.tar.gz](https://github.com/zilliztech/milvus-backup/files/10390478/artifacts-backup-test-25-pytest-logs.tar.gz)

**Comment by wayblink** [comment]
Should be fixed in #62
